assets/js/components/notifications/dashboard-core-site-alerts.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * DashboardCoreSiteAlerts component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import Notification from 'GoogleComponents/notifications/notification';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getNotifications,
+	acceptNotification,
+	dismissNotification,
+} from './site';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+class DashboardCoreSiteAlerts extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			notifications: false,
+		};
+	}
+
+	async componentDidMount() {
+		const notifications = await getNotifications().catch( ( e ) => {
```

Comment:
For `async/await` code it's usually the norm to do try/catch blocks instead of `catch`. Any reason to prefer catch here?
```suggestion
		try {
			const notifications = await getNotifications();
			this.setState( { notifications } );
		} catch ( err ) {
			// eslint-disable-next-line no-console
			console.warn( 'Error caught while fetching notifications', JSON.stringify( err ) );
		}
```

I dunno what `notifications` get set to when this fails, but it seems off to set notifications if there was an error fetching them too.